mmv1/templates/terraform/examples/bigquery_reservation_capacity_commitment_basic.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_bigquery_capacity_commitment" "<%= ctx[:primary_resource_id] %>" {
+	location   = "us-west1"
+	slot_count = 100
+	plan       = "FLEX"
+}
+
+resource "time_sleep" "wait_60_seconds" {
+	depends_on = [google_bigquery_capacity_commitment.<%= ctx[:primary_resource_id] %>]
```

Comment:
this basically makes a minute delay after creating? That's the only way to delete afterwards? Is this eventually consistent?